tests/foreman/ui/test_hostunification.py

```diff
@@ -67,32 +93,33 @@ def test_positive_register_host_via_ak(self):
         @assert: Hosts registered via activation key appears under 'All hosts'
         as well as under content-hosts
 
-        @caseautomation: notautomated
-
         @CaseLevel: System
         """
+        org = entities.Organization().create()
+        env = entities.LifecycleEnvironment(organization=org).create()
+        content_view = entities.ContentView(organization=org).create()
+        activation_key = entities.ActivationKey(
+            environment=env,
+            organization=org,
+        ).create()
+        setup_org_for_a_rh_repo({
+            'product': PRDS['rhel'],
+            'repository-set': REPOSET['rhst7'],
+            'repository': REPOS['rhst7']['name'],
+            'organization-id': org.id,
+            'content-view-id': content_view.id,
+            'lifecycle-environment-id': env.id,
+            'activationkey-id': activation_key.id,
+        })
+        with VirtualMachine(distro='rhel71') as vm:
+            vm.install_katello_ca()
+            result = vm.register_contenthost(org.label, activation_key.name)
+            self.assertEqual(result.return_code, 0)
+            with Session(self.browser) as session:
+                session.nav.go_to_select_org(org.name)
+                self.assertIsNotNone(self.hosts.search(vm.hostname))
+                self.assertIsNotNone(self.contenthost.search(vm.hostname))
 
-    @stubbed()
-    @run_only_on('sat')
-    @tier3
-    def test_positive_validate_org(self):
```

Comment:
All the hosts we register via rhsm have organization specified, for example [here](https://github.com/SatelliteQE/robottelo/pull/3577/files#diff-25c1fd038f0ca18fe9cded0814312d79R74) or [here](https://github.com/SatelliteQE/robottelo/pull/3577/files#diff-25c1fd038f0ca18fe9cded0814312d79R116). So basically this test is already covered in any other host registration test. And it's not an option not to specify org in other tests - we won't be able to see the host on ui, as the user is forced to select organization to see hosts :)